backend/app/services/internal/file_upload.py

```
@@ -0,0 +1,38 @@
+        size: int,
+    ):
+        try:
+            return UserFileRepository().create_file(
+                user_id=user_id,
+                type=type,
+                name=name,
+                key=key,
+                size=size,
+            )
+        except Exception as e:
+            logger.error(f"Failed to add file to database: {str(e)}")
```

Ruff (0.14.13)

Use explicit conversion flag

Replace with conversion flag

(RUF010)